src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx

```diff
@@ -14,37 +14,55 @@ redirects:
   - /docs/accounts/accounts-billing/new-relic-one-pricing-billing
   - /docs/licenses/license-information/faq/new-relic-one-pricing-plan-frequently-asked-questions
   - /docs/licenses/license-information/faq/frequently-asked-questions
+  - /docs/accounts/accounts-billing/new-relic-one-pricing-billing/billing-ui
+  - /docs/accounts/accounts-billing/general-account-settings/introduction-account-settings
 ---
 
-New Relic has two primary billing factors: ingested data and billable users. In this doc, we'll explain how pricing works, what you get for free, and other details. 
+import accountsUsageUi from 'images/accounts_screenshot-full_usage-ui.webp'
+
+The New Relic pricing model is simple and relies on two primary billing factors: how much data you ingest and how many billable users you have. We'll explain where to find your billing UI, how our pricing plan works, what you get for free, and other details. 
 
 <Callout variant="tip">
-This doc is about our newer usage-based pricing model, and not our original pricing model. You're probably on our newer pricing model, but if you're not sure, see [Pricing models](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-pricing-models).
+A small number of older New Relic customers are on an older pricing model. For docs on that, see [Pricing models](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-pricing-models).
 </Callout>
 
-## Where to start [#start]
+## Not yet a customer? [#start]
+
+If you're not yet a New Relic customer and want an overview of how our pricing works, we recommend checking out our main [pricing page](https://newrelic.com/pricing). If you want more in-depth details about how our pricing works, keep reading. 
+
+## The billing and usage UI [#usage-ui]
+
+If you're a New Relic customer, the best way to understand your billing is to go to the usage UI ([direct link to UI](https://one.newrelic.com/admin-portal/consumption-usage)). To find that, go to **[one.newrelic.com](https://one.newrelic.com)**, click the [user menu](/docs/accounts/accounts-billing/general-account-settings/intro-account-settings), then click **Administration**. You'll land on the **Plan & usage** UI page, which shows an overview of your usage.
```

Comment:
Do we need the (direct link to UI) part? Can that link just be the "usage UI"?

Reply:
i don't really know. sometimes i think it's unclear if that's just linking to another doc or what